repository/model/settings.go

```diff
@@ -0,0 +1,99 @@
+package model
+
+import (
+	"encoding/json"
+	"fmt"
+
+	reperr "release-manager/repository/errors"
+	svcmodel "release-manager/service/model"
+)
+
+const (
+	orgNameKey           = "organization_name"
+	slackTokenKey        = "slack_token"
+	githubTokenKey       = "github_access_token"
+	defaultReleaseMsgKey = "default_release_msg"
+)
+
+type Setting struct {
```

Comment:
In the database, settings are saved in a table with two columns: key and value. The value is stored as JSON. This approach enables easy addition of more setting options in the future